mne/io/eeglab/eeglab.py

```diff
@@ -354,6 +690,13 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
         self._create_event_ch(events, n_samples=eeg.pnts)
 
         # read the data
+        if isinstance(eeg, Bunch):
+            # if hdf5 data was read in, then file name might be encoded
+            # as array of ascii values of characters
+            ascii_check = _check_for_ascii_filename(eeg, input_fname)
+            if ascii_check[0]:
+                eeg.data = ascii_check[1]
```

Comment:
Can't you put thsi is `_get_eeg_data`? Then it doesn't need to be replicated here and in the Epochs reader.